### PR TITLE
To-do List 

### PR DESCRIPTION
**Pull request by eric2umeh**
- Add Webpack, Webserver, HTML Webpack Plugin, Style/CSS Loader
- Create a dynamic todo list
- linter error check

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      {
+        test: /\.css$/i,
+        use: ['style-loader', 'css-loader'],
+      },
+    ],
+  },
+  optimization: {
+    runtimeChunk: 'single',
+  },
+};
```
Comment (Gambit142) [inline webpack.config.js]: - [ ] When I start the server on port 8080, I get this error message:

image

You can resolve this by implementing the solution in this [article](https://webpack.js.org/configuration/mode/) 👍

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    const LocalStore = {
+      index,
+      description: taskInput.value,
+      completed,
+    };
+    description.push(LocalStore);
+    localStorage.setItem('todoList', JSON.stringify(description));
+    displayTodo();
+  }
+});
```
Comment (Gambit142) [inline src/index.js]: - [ ] Great work implementing this. For this milestone, the requirements state that you should create an array of simple to-do tasks and render it. Kindly implement it this way for this milestone. 👍 💻 

image

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @eric2umeh 👋, 

Good job so far!

## Highlights
- All linters checks passed ✔️ 
- `Webpack` was used to bundle JavaScript code ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: webpack.config.js (hunk 1 above)
Comment (eric2umeh) [reply webpack.config.js]: Bless you for this article @Gambit142 . Great help and review

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

### Hey @eric2umeh 👋

Great work on implementing the requirements. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Requirements
✔️ Following Linters

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**